source/_integrations/enigma2.markdown

```diff
@@ -14,8 +14,92 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-The `enigma2` platform allows you to control a Linux based set-top box which is running [Enigma2](https://github.com/oe-alliance/oe-alliance-enigma2) with the OpenWebif plugin installed.
+The **Enigma2** {% term integration %} allows you to control a Linux based set-top box which is running [Enigma2](https://github.com/oe-alliance/oe-alliance-enigma2) with the OpenWebif plugin installed.
 
-[OpenWebif](https://github.com/E2OpenPlugins/e2openplugin-OpenWebif) is an open source web interface for Enigma2 based set-top boxes.
+[OpenWebif](https://github.com/E2OpenPlugins/e2openplugin-OpenWebif) is an open-source web interface for Enigma2 based set-top boxes.
+
+### Prerequisites
+
+Your device needs to have the OpenWebif plugin installed. On most devices it is installed by default, if not, it is available via the Plugins menu within your Enigma2 distribution.
+
+Please beware that the OpenWebif setting "Require client cert for HTTPS" is not supported.
 
 {% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+    description: "The IP address or hostname of your device."
+Port:
+    description: "The port of the OpenWebif service running. Defaults to 80."
+Username:
+    description: "The username, if HTTP(S) Authentication is enabled."
+Password:
+    description: "The password, if HTTP(S) Authentication is enabled."
+Uses an SSL certificate:
+    description: "Whether HTTPS is enabled."
+Verify SSL certificate:
+    description: "Whether the SSL certificate should be verified."
+{% endconfiguration_basic %}
+
+## Configuration options
+
+The integration provides the following configuration options:
+
+{% configuration_basic %}
+Turn off to deep standby:
+    description: "Shuts the device down (called Deep Standby) on turning off the device."
+Bouquet to use as media source:
+    description: "Sets the bouquet to use for the source list."
+{% endconfiguration_basic %}
+
+{% warning %}
+When the device is in Deep Standby, it can no longer be reached! Turning on the device is only possible via:
+
+- Wake on LAN
+- Power button on the device
+- Remote control
+{% endwarning %}
```

Comment:
```suggestion
```
I would move this information right next to the setting information. So that people who don't read any further still see it, before they change the setting.